### ISSUE TITLE
Problem with manufacturers 

### ISSUE THREAD

**Issue by Raito00** [faktiva/prestashop-clean-urls#79]
I have problem with manufactures url - 404 error

in SEO i have: manufactures/{rewrite}

PS. 1.6.1.11
php 7.0

**Comment by drAlberT** [comment]
It should work

Need more info plz

**Comment by Raito00** [comment]
Here is test server
https://178.62.21.158

**Comment by Raito00** [comment]
it take link from manufacture name not for rewrite link

i check theme manufacturer-list.tpl

`href="{$link->getmanufacturerLink($manufacturer.id_manufacturer, $manufacturer.link_rewrite)|escape:'html':'UTF-8'}"`

there is id_manufacturer why it generate link from name ?

or i`m wrong ...

BR
R

**Comment by Raito00** [comment]
List of the manufacurer working
https://178.62.21.158/sv/manufactures

but not working if click on specific manufacturer :(

**Comment by drAlberT** [comment]
`$link->getmanufacturerLink()` will generate the URL starting from id and link_rewrite .. it is ok

Double check that the Link.php override is installed correctly .. moreover please take a screenshot of your SEO & URLs config

**Comment by Raito00** [comment]
I allready deleted all files in override and cache clas-index then install module again. The same problem.
Maybe better You can log in and see all setings in website
I have send to prestashop-seo-tk@faktiva.com all login details

**Comment by drAlberT** [comment]
Yes .. but not today ;)

**Comment by drAlberT** [comment]
please set default language to English .. I have some troubles navigating the BO ;)

**Comment by drAlberT** [comment]
There is another URL cleaner module installed ...

**Comment by Raito00** [comment]
Yes i was testing other modules. You can change what You need to check
where is problem

**Comment by drAlberT** [comment]
If I change admin language to English I still have SE selected ..

**Comment by Raito00** [comment]
I double check there is set English language ..

**Comment by drAlberT** [comment]
It looks working to me now .. can you check plz?

**Comment by Raito00** [comment]
No! 
manufacturers still 404 error

**Comment by drAlberT** [comment]
I have no access to this section on the BO .. need to check it

**Comment by Raito00** [comment]
Try now

**Comment by maxclee** [comment]
Hi, please explain what was wrong , I have the same problem  ... 
thanks

**Comment by drAlberT** [comment]
> There is another URL cleaner module installed ...

**Comment by maxclee** [comment]
Hi, thanks for answer 
I have no other url cleaner >

**Comment by maxclee** [comment]
@drAlberT I have installed last Faktiva Clean URLs v1.2.2 
What more can cause problems with manufactures links?

**Comment by drAlberT** [comment]
I don't know. I should have some way to take a look ..
